README.md

```diff
@@ -13,23 +13,24 @@ REST API for managing cache control information for pages within the legacy CMS
-| MONGODB_DATABASE                   | cache                                                      | The MongoDB database                                                                                                |
-| MONGODB_COLLECTIONS                | CacheTimesCollection:cachetimes                            | The MongoDB collections                                                                                             |                           
-| MONGODB_REPLICA_SET                |                                                            | The name of the MongoDB replica set                                                                                 |
-| MONGODB_ENABLE_READ_CONCERN        | false                                                      | Switch to use (or not) majority read concern                                                                        |
-| MONGODB_ENABLE_WRITE_CONCERN       | true                                                       | Switch to use (or not) majority write concern                                                                       |
-| MONGODB_CONNECT_TIMEOUT            | 5s                                                         | The timeout when connecting to MongoDB (`time.Duration` format)                                                     |
-| MONGODB_QUERY_TIMEOUT              | 15s                                                        | The timeout for querying MongoDB (`time.Duration` format)                                                           |
-| MONGODB_IS_SSL                     | false                                                      | Switch to use (or not) TLS when connecting to mongodb                                                               |
-| GRACEFUL_SHUTDOWN_TIMEOUT          | 5s                                                         | The graceful shutdown timeout in seconds (`time.Duration` format)                                                   |
-| HEALTHCHECK_INTERVAL               | 30s                                                        | Time between self-healthchecks (`time.Duration` format)                                                             |
-| HEALTHCHECK_CRITICAL_TIMEOUT       | 90s                                                        | Time to wait until an unhealthy dependent propagates its
+| Environment variable         | Default                         | Description                                                       |
+|------------------------------|---------------------------------|-------------------------------------------------------------------|                
+| ZEBEDEE_URL                  | localhost:8082                  | Zebedde host address and port for authentication                  |
```

Comment:
Small one, but I'd move this to the bottom of the table, as normally the first one tends to be the BIND_ADDR in the other ONS repos I've seen.

Reply:
That's also not exactly the default address you've defined in the config. In the config it has the `http` protocol. Given that the mongoDB bind address does not have `http://`, maybe that's the way to go.